### PR TITLE
[Integration Test][ome_application_network_settings] - All features

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: tests/integration/targets/ome_application_network_settings/tests/ansible_doc.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- name: ansible-doc
+  command: ansible-doc dellemc.openmanage.ome_application_network_settings
```
Comment (ShivamSh3) [inline tests/integration/targets/ome_application_network_settings/tests/ansible_doc.yaml]:
```suggestion
  ansible.builtin.command: ansible-doc dellemc.openmanage.ome_application_network_settings
```

File: tests/integration/targets/ome_application_network_settings/tests/ansible_doc.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  register: ansible_doc_hash
+
+- local_action: copy content={{ansible_doc_hash.stdout_lines[2:] | to_nice_yaml}} dest={{ role_path }}/files/ansible_doc.txt
+  register: ansible_doc
+  check_mode: true
+  diff: true
+
+- name: Verify if doc changed
+  assert:
```
Comment (ShivamSh3) [inline tests/integration/targets/ome_application_network_settings/tests/ansible_doc.yaml]:
```suggestion
  ansible.builtin.assert:
```

File: tests/integration/targets/ome_application_network_settings/tests/invalid_common.yaml [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+          universal_timeout: 30
+      ignore_errors: true
+      register: invalid_ca_path
+
+    - name: Verify task status - Negative - Invalid ca_path
+      ansible.builtin.assert:
+        that:
+          - invalid_ca_path.failed
+          - not invalid_ca_path.changed
+          # - 'invalid_ca_path.msg == "[Errno 2] No such file or
```
Comment (ShivamSh3) [inline tests/integration/targets/ome_application_network_settings/tests/invalid_common.yaml:110]: remove comments

File: tests/integration/targets/omem_application_network_settings/tests/invalid_common_omem.yaml [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+          universal_timeout: 30
+      ignore_errors: true
+      register: invalid_ca_path
+
+    - name: Verify task status - Negative - Invalid ca_path
+      ansible.builtin.assert:
+        that:
+          - invalid_ca_path.failed
+          - not invalid_ca_path.changed
+          # - 'invalid_ca_path.msg == "[Errno 2] No such file or
```
Comment (ShivamSh3) [inline tests/integration/targets/omem_application_network_settings/tests/invalid_common_omem.yaml:110]: remove comments

Comment (Saksham-Nautiyal) [reply tests/integration/targets/omem_application_network_settings/tests/invalid_common_omem.yaml:110]: commented due to module defect

**Review by rajshekarp87 — APPROVED** [review]

**Review by ShivamSh3 — APPROVED** [review]

**Pull request merged**